content/en/docs/reference/networking/virtual-ips.md

```diff
@@ -394,6 +394,43 @@ NAME              PARENTREF
 2001:db8:1:2::a   services/kube-system/kube-dns
 ```
 
+This new feature also allow users to dynamically define the `service-cluster-ip-range` using
```

Comment:
we tend to call `--service-cluster-ip-range` a flag for a command line tool,
but we seldom use the spelling `service-cluster-ip-range` which is confusing.
The original text on line 373 is confusing as well.

Reply:
done